addons/l10n_ua/data/template/account.account-ua_psbo.csv

```diff
@@ -208,14 +205,14 @@
 "ua_psbp_6415","6415","Розрахунки по військовому збору","liability_payable","True",""
 "ua_psbp_6416","6416","Розрахунки за акцизом","liability_payable","True",""
 "ua_psbp_642","642","Розрахунки по обов’язкових платежах","liability_payable","True",""
-"ua_psbp_6431","6431","Податкові зобов’язання","liability_current","False",""
-"ua_psbp_6432","6432","Податкові зобов’язання непідтверджені","liability_current","False",""
-"ua_psbp_6433","6433","Коригування податкових зобов’язань","liability_current","False",""
-"ua_psbp_6434","6434","Податкові зобов’язання (роздріб)","liability_current","False",""
-"ua_psbp_6435","6435","Умовний продаж","liability_current","False",""
-"ua_psbp_6441","6441","Податковий кредит","liability_current","False",""
-"ua_psbp_6442","6442","Податковий кредит непідтверджений","liability_current","False",""
-"ua_psbp_6443","6443","Коригування податкового кредиту","liability_current","False",""
+"ua_psbp_6431","6431","Податкові зобов’язання","liability_current","True",""
+"ua_psbp_6432","6432","Податкові зобов’язання непідтверджені","liability_current","True",""
```

Comment:
These accounts are used in tax repartition lines. 
It's quite incompatible to have bank reconciliation on tax provision accounts.
Please see "reconcile = true" as : visible in the bank reco widget when accounting is installed.